nitransforms/tests/data/regressions/robust_register.lta

```diff
@@ -1,30 +1,28 @@
-# transform file /home/oesteban/tmp/fmriprep-ds005/fprep-work/fmriprep_wf/single_subject_01_wf/anat_preproc_wf/surface_recon_wf/fsnative2t1w_xfm/T1_robustreg.lta
-# created by oesteban on Sat Mar 14 19:28:37 2020
-
```

Comment:
Does this new file fail prior to #65?

Reply:
Yes, I have updated both because I had messed up when filing #64. The attached ITK file I believe is the inverse. To be sure, I searched again for both (LTA, ITK) from the robust register node and subsequent nodes and replaced them.